pages/en/tokenomics.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Tokenomics of The Graph Network
+navTitle: Tokenomics
+description: The Graph Network is incentivized by powerful tokenomics. Here’s how GRT, The Graph’s native work utility token works.
```

Comment:
Just noticed this. We don't do anything with a field called `description` at the moment; what are you trying to do?

Reply:
This was an SEO description/subtitle. Any other way to include this for SEO purposes?

Reply:
Handled in https://github.com/graphprotocol/docs/pull/254/commits/a556f8d2e2e6a5c9ed41503e21921a65b0728bd7